### PR TITLE
updated April history

### PR DESCRIPTION
**Pull request by mportman12**
Update April history

### CODE REVIEW

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 1]
```diff
@@ -30,6 +30,24 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 // .Other updates
 // * TODO
 
+[discrete]
+[[april-2023]]
+== April 2023
+
+.Glossary entries
+* xref:apache-web-server[Apache web server]: Updated to remove extraneous IdM definitions
+* xref:certificate-authority[certificate authority]: Updated to remove extraneous IdM definitions
```
Comment (bredamc) [inline supplementary_style_guide/introduction/whats-new.adoc:39]:
```suggestion
* xref:certificate-authority[certificate authority]: Renamed from "certificate authorities", and updated to remove extraneous IdM definitions
```

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 2]
```diff
@@ -30,6 +30,24 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
...
+* xref:domain-controller[domain controller]: Updated to remove extraneous IdM definitions
+* xref:kerberos-protocol[Kerberos protocol]: Updated to remove extraneous IdM definitions
+* xref:kerberos-realm[Kerberos realm]: Updated to remove extraneous IdM definitions
+* xref:posix-attributes[POSIX attributes]: Updated to remove extraneous IdM definitions
+* xref:web-server[web server]: Updated to remove extraneous IdM definitions
+
+.Other updates
+* Added xref:non-breaking-spaces[non-breaking spaces] between "Red" and "Hat" within the guide.
```
Comment (bredamc) [inline supplementary_style_guide/introduction/whats-new.adoc:47]:
```suggestion
* Added a xref:non-breaking-spaces[non-breaking space] between "Red" and "Hat" in each occurrence within the guide.
```

File: supplementary_style_guide/introduction/whats-new.adoc [hunk 3]
```diff
@@ -30,6 +30,24 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
...
+* xref:shortdesc[Short descriptions]: Added guidance on writing short descriptions and abstracts.
```
Comment (bredamc) [inline supplementary_style_guide/introduction/whats-new.adoc:48]:
```suggestion
* xref:shortdesc[Short descriptions]: Added guidance on writing _short descriptions_ (also known as _abstracts_).
```

**Review by bredamc — COMMENTED** [review]
Just a few very minor comments.

**Review by bredamc — APPROVED** [review]
LGTM :)

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**